examples/persistence_transform.py

```diff
@@ -0,0 +1,52 @@
+"""
+========================================
+Persistence Transform in MapSequence
+========================================
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+
+.. note::
+   This example requires ``mpl_animators`` to be installed.
+"""
+
+import matplotlib.cm as cm
+import matplotlib.colors as colors
+import matplotlib.pyplot as plt
+import numpy as np
+
+import sunpy.map
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapSequence using the AIA_193_CUTOUT sample data.
+# To create a MapSequence, we can call Map directly but add in a
+# keyword to output a MapSequence instead.
+
+aiamapseq = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE, cube=True)
+
+###############################################################################
+# For a data set consisting of N images with intensity values I(x,y,t),
```

Comment:
Maybe this should be rendered as math text

Reply:
> I think we need to change the docs/config.py so the gallery will have animations.

Hmm ... i will go through that 👍